templates/focus-ios.xliff

```diff
@@ -367,6 +379,10 @@
         <source>Save</source>
         <note>Save button label</note>
       </trans-unit>
+      <trans-unit id="SearchEngine.addEngineError">
+        <source>That didn't work. Try replacing the search term with this: %s.</source>
+        <note>Label for error state when entering an invalid search engine URL.</note>
```

Comment:
Nit: I'd just add what %s is replaced by here (as a general rule, we always ask for a comment explaining placeholders)

Reply:
This actually isn't a placeholder - the user here is giving a URL for a search engine, and if it fails, we suggest that they replace the search term with the string "%s" since they are constructing a URL.

Reply:
I see. Still probably a good idea just to call that out in a simple way in comments (localizers can get easily confused, lol). I would suggest to put something like:  ""%s" will be replaced by a suggestion of another search term"
something simple along those lines :)